crates/nu-protocol/src/engine/state_working_set.rs

```diff
@@ -967,7 +966,7 @@ impl<'a> StateWorkingSet<'a> {
             }
         }
 
-        for block in &self.permanent_state.blocks {
+        for block in self.permanent_state.blocks.iter() {
```

Comment:
Sorry I'm not really sure, why are you using `iter()` here, but removing `iter()` call on `crates/nu-cli/src/completions/completer.rs:L127`?

Reply:
I'm not sure either! It probably just got caught up in some kind of intermediate change and I meant to change it back.